### ISSUE TITLE
Default gateway

### ISSUE THREAD

**Issue by squaremo** [weaveworks/docker-plugin#25]
I'm not sure in what circumstances we want to provide a default gateway; perhaps always, since we can usually assume we'll be the only interface. But is it correct to do so?

**Comment by awh** [comment]
@squaremo can we close this given Docker's `docker_gwbridge` functionality?

**Comment by squaremo** [comment]
Did you figure out what determines whether or not you get an interface on the `docker_gwbridge`?

**Comment by awh** [comment]
No, good thinking - we should make sure we understand that before relying on it.
